### DISCUSSION TITLE
ERROR   -  Config value 'theme': The theme 'docskimmer' does not appear to have a configuration file.

### DISCUSSION THREAD

**Discussion by pawamoy** [hfagerlund/mkdocs-docskimmer#18] [General]
Hello, is this theme still maintained?

I'm building a [gallery of MkDocs themes](https://pawamoy.github.io/mkdocs-gallery/) and am getting this error when building:

```
ERROR   -  Config value 'theme': The theme 'docskimmer' does not appear to have a configuration file. Please upgrade to a current version of the theme.

Aborted with 1 configuration errors!
```

**Comment by hfagerlund** [comment]
@pawamoy -

The [latest version of this theme (v0.4.0)](https://github.com/hfagerlund/mkdocs-docskimmer#version-compatibility) was tested with [MkDocs version 1.0.4](https://www.mkdocs.org/about/release-notes/#version-104-2018-09-07). 
(The [latest release of MkDocs at the time of writing](https://github.com/mkdocs/mkdocs/releases/tag/1.5.3) is v1.5.3.)

I will try to take a look, but if you find the time before I do, please feel free to [make a contribution to the docSkimmer theme project](https://github.com/hfagerlund/mkdocs-docskimmer/fork). Thanks.

**Comment by pawamoy** [comment]
Thank you! My plate is a little full right now, so I will probably not contribute any time soon. I was mainly reporting to see if something could be done, or if I should remove the theme from the gallery :slightly_smiling_face: For now I've removed it (since it doesn't build at all), but if you get it working again for latest MkDocs version, let me know and I'll reinclude it immediately :slightly_smiling_face:

**Comment by hfagerlund** [comment]
@pawamoy -

Without investigating the [changes to MkDocs since 2018](https://www.mkdocs.org/about/release-notes/) (when the latest version of this theme was released), it is impossible to say whether this issue could be remedied with a simple configuration change, or if it is a request for a new version of the theme.

**Comment by hfagerlund** [comment]
After testing using Docker, I cannot reproduce the reported issue - [running the build using the latest versions of docSkimmer and MkDocs](https://github.com/hfagerlund/docker-for-docskimmer) works fine for me - ie. builds without any errors.

**Reply by oprypin** [reply]
@hfagerlund Hmm it seems like you're bypassing the `pip install` step and that's what makes the difference.

https://github.com/hfagerlund/docker-for-docskimmer/blob/aa418ee766fd4e3c84a7bec19370968059236c0e/Dockerfile#L10
versus 
`pip install git+https://github.com/hfagerlund/mkdocs-docskimmer.git`

mkdocs_theme.yml is present in the source tree but is not present in the built package

due to manifest stuff
https://github.com/hfagerlund/mkdocs-docskimmer/blob/3141195cffc760703c3855c3fa22a7e63fa63053/MANIFEST.in#L1

**Reply by hfagerlund** [reply]
@oprypin 

Thanks for the detailed follow-up. I will take a look.

**Reply by hfagerlund** [reply]
@oprypin 

I have [created an experimental branch for the fix](https://github.com/hfagerlund/mkdocs-docskimmer/pull/19), and here is [my testing](https://github.com/hfagerlund/docker-for-docskimmer/compare/v0.1.0...v0.1.1) - the (revised) build works for me. 

If you have time, please let me know whether you are still seeing an error - thanks.

**Reply by oprypin** [reply]
@hfagerlund Yes that looks great, and the testing is rock-solid 🙂
You can merge it already, it'll work for sure. I'll test tomorrow.

**Reply by hfagerlund** [reply]
@oprypin

Many thanks - done ([merged](https://github.com/hfagerlund/mkdocs-docskimmer/commit/2800af91a7cd1f6e7d399ed5321d039ca6b2467a)).
